Fixs the SearchString parameter for Get-AzureRmADServicePrincipal

AdamCLarsen opened the pull request:
The service Principal for Azure Kay Vault has spaces.

azurecla commented:
Hi __@ChrisCompy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

ChrisCompy commented:
Jaime worked the same issue separately.  His is already on the way.  closing this request out.

AdamCLarsen committed:
Fixs the SearchString parameter for Get-AzureRmADServicePrincipal

The service Principal for Azure Kay Vault has spaces.

fhokholdMSFT commented on articles/key-vault/key-vault-ovw-storage-keys.md:
```diff
@@ -102,7 +102,7 @@ Key Vault needs permissions to *list* and *regenerate* keys for a storage accoun
     
      or
      
-    `Get-AzureRmADServicePrincipal -SearchString "AzureKeyVault"`
```
Hi Adam, I think that we should remove this line completely to be honest, I think it just creates confusion as to how to get the Id. Also, people can add names and so there could be multiple "Azure Key Vault" just like there is multiple "AzureKeyVault". Let me know what you think.

BrucePerlerMS replied:
@AdamCLarsen, @fhokholdMSFT - thanks for talking this through. This is another important way our content gets improved.
~ Bruce

AdamCLarsen replied:
@fhokholdMSFT I agree, especially if I was to be scripting this.  Then I really should only use the GUID.

It wasn't clear to me just reading it, that the GUID was an Azure global GUID.  And not until I got the sample completed and saw it in the portal, did I realized what that step was actually doing.

AdamCLarsen committed:
Changing the distributions and use around the Azure Key Vault Service Principal

It was initially unclear what the GUID was the global service principal for Azure Key Vault.  This updates the sample code to remove some of the ambiguity around what the Object It to use.

fhokholdMSFT reviewed (approved):
Hi Adam, I think the changes look good. Thanks for your help.
